brewtils/rest/client.py

```diff
@@ -943,3 +944,71 @@ def get_tokens(self, username=None, password=None):
             self.session.headers["Authorization"] = "Bearer " + self.access_token
 
         return response
+
+    @enable_auth
+    def get_topic(self, topic_id, **kwargs):
```

Comment:
Not sure you need the `**kwargs` here. Only the `topic_id` is passed forward and the API doesn't access additional parameters.

Reply:
Removed kwargs